RELEASE-NOTES.md

```diff
@@ -1,5 +1,25 @@
 # Cumulative Release Notes for the Annotated Data Model
 
+## ???
+
+### [ROS-218](https://basistech.atlassian.net/browse/ROS-218) Entity-level sentiment is a list
+
+Entity-level sentiment used to return a single `CategorizerResult`.
+After this change it returns a `List<CategorizerResult>` in confidence
+ranked order.  Currently there will be three results, one for each of
+"neg", "neu", "pos".  `ResolvedEntity#getSentiment` is already
+deprecated because of ROS-43.  It still returns the top result only.
+`Entity#getSentiment()` is the new method that returns the list.
+
+### [ROS-43](https://basistech.atlassian.net/browse/ROS-43) Combine EntityMentions and ResolvedEntity
+
+TODO
```

Comment:
I'd like to see an explanation here of what folks with old code and old json
files might want to do to upgrade.  At minimum, mention specifically what
was deprecated and moved, new classes, etc.  (I can give it a go; just let
me know...)